### DISCUSSION TITLE
nvm windows exec doesn't work on windows 10

### DISCUSSION THREAD

**Discussion by ahmedsalihh** [coreybutler/nvm-windows#585] [Support] [answered]
I've installed nvm windows on my windows 10. but I can't run exec or run command.
![image](https://user-images.githubusercontent.com/13359680/99299957-08d24680-285d-11eb-9052-4acc2b0cdcfc.png)
when I tried this command on ubuntu it works but not on windows. I need to run node version 6 for angular 4 project which needs nodejs 6 and lts version of nodejs for new react project.

Is there any solution about this or am I missing something ?

**Comment by coreybutler** [comment, marked as answer]
That's because you're trying to use [nvm](https://github.com/nvm-sh) commands in NVM for Windows. They're two completely different projects (see the README).

**Reply by ahmedsalihh** [reply]
is there anything like exec or run in nvm for windows ?

**Reply by coreybutler** [reply]
No.

Those commands serve the purpose of preconfiguring/rearranging the local environment for a particular process. The original nvm project relies on a shim to do this, which can cause a significant number of problems on Windows systems (with antivirus, permissions, etc). If you look at this project's [roadmap](https://github.com/coreybutler/nvm-windows/projects/1), you'll see I'm planning to add a hook system (project "rt"). This would work in a manner similar to git hooks, serving the same purpose as `exec` and `run`, but without requiring a shim.
